wolfcrypt/src/ecc.c

```diff
@@ -7777,6 +7812,7 @@ int wc_ecc_check_key(ecc_key* key)
     #endif
 
     /* SP 800-56Ar3, section 5.6.2.3.3, process step 1 */
+    /* SP 800-56Ar3, section 5.6.2.3.4, process step 1 */
```

Comment:
Did the section number change or do both apply? Perhaps consolidate tis and the one at 7845?

Reply:
The code covers both section. dot3 is the full validation, and dot4 is the partial validation. When you are the recipient of a public key, you cannot perform a full validation.